components/action-card/types.ts

```diff
@@ -1,4 +1,4 @@
-export type IActionIconVariant = 'deposit' | 'stake' | 'borrow' | 'supply' | 'sort';
+export type IActionIconVariant = 'deposit' | 'stake' | 'borrow' | 'supply' | 'sort' | 'debitCard' | 'crypto' | 'coinBase';
```

Comment:
I don't understand why we need this. There are lots of variants defined in the types that it confuses me. This will be refactored. Leave it for now though so it doesn't affect my work